rules/privatization/src/Rector/Property/PrivatizeFinalClassPropertyRector.php

```diff
@@ -106,7 +106,7 @@ private function isPropertyVisibilityGuardedByParent(Property $property, Class_
     }
 
     /**
-     * @return class-string[]
+     * @return class-string[]|bool[]
      */
     private function getParentClasses(Class_ $class): array
     {
```

Comment:
It would be better to modify code bellow to return only strings.

Reply:
fixed.